03/app.js

```diff
@@ -1,17 +1,19 @@
-//DZIAŁA
+//BŁAD linia 14
 const input = document.querySelector('input'); //ok
 
 const newItem = document.querySelector('.images-list__item--prototype').cloneNode(true);
 newItem.style.display = 'block';
 
 //nasłuchiwanie na 'change'
 input.addEventListener('change', function(e) {
-    const file = e.target.value;
-    // console.log(e.target.value);
+    const file = e.target.files[0]; // do wyjaśnienia
+    console.log(file);
+    console.log(e.target.value);
     
     // dodaję do tablicy pobrane pliki
     const fileList = [];
-    const fileDownloaded = file;
+    const fileDownloaded = file.type.includes['image']; // BŁąD undefined
+    console.log(fileDownloaded);
```

Comment:
To zadanie jest jeszcze nie dokończone

Reply:
Przepraszam, nie zauważyłam, że nie było dokończone. Teraz jest już ok zrobione.